frame/proxy/src/benchmarking.rs

```diff
@@ -86,7 +86,7 @@ benchmarks! {
 		let call: <T as Config>::Call = frame_system::Call::<T>::remark { remark: vec![] }.into();
 	}: _(RawOrigin::Signed(caller), real, Some(T::ProxyType::default()), Box::new(call))
 	verify {
-		assert_last_event::<T>(Event::ProxyExecuted(Ok(())).into())
+		assert_last_event::<T>(Event::ProxyExecuted{result: Ok(())}.into())
```

Comment:
```suggestion
		assert_last_event::<T>(Event::ProxyExecuted{ result: Ok(()) }.into())
```
Same for rest of file